src/DocumentationBrowserViewExtension/DocumentationBrowserViewModel.cs

```diff
@@ -19,7 +19,41 @@ public class DocumentationBrowserViewModel : NotificationObject, IDisposable
         private const string BUILT_IN_CONTENT_FILE_NOT_FOUND_FILENAME = nameof(Resources.FileNotFound) + ".html";
         private const string BUILT_IN_CONTENT_NO_CONTENT_FILENAME = "NoContent.html";
         private const string SCRIPT_TAG_REGEX = @"<script[^>]*>[\s\S]*?</script>";
+        private const string DPISCRIPT = @"<script> function getDPIScale()
+        {
+            var dpi = 96.0;
+            if (window.screen.deviceXDPI != undefined)
+            {
+                dpi = window.screen.deviceXDPI;
+            }
+            else
+            {
+                var tmpNode = document.createElement('DIV');
+                tmpNode.style.cssText = 'width:1in;height:1in;position:absolute;left:0px;top:0px;z-index:99;visibility:hidden';
+                document.body.appendChild(tmpNode);
+                dpi = parseInt(tmpNode.offsetWidth);
+                tmpNode.parentNode.removeChild(tmpNode);
+            }
+
+            return dpi / 96.0;
+        }
+
+        function adaptDPI()
+        {
+            var dpiScale = getDPIScale();
+            document.body.style.zoom = dpiScale;
 
+            var widthPercentage = ((100.0 / dpiScale)-5).toString() + '%';
+            document.body.style.width = widthPercentage;
+        }
+        adaptDPI() </script>";
+
+        private const string globalimgStyle = @"<style> img{
+        min-width:300px;
+        max-width:450px;
+        width:100%;
+                }
+        </style>";
```

Comment:
Just a note about this. Defining the style like this, without CSS class, will affect any `<img>` tag, including those that are included by library developers in their html help files. That means in practice we are putting restrictions to their pages, which may be a good thing taking into account the scrollbar issue, but could be limiting if they wanted to show a really big image.

Reply:
that's a good point, I guess for max flexibility I can just add this style to each of our pages that uses images. Not too much work, what do you think?

Reply:
@mjkkirschner That could work as well

Reply:
done.